spec/system/shared/proposals_steps_examples.rb

```diff
@@ -149,6 +152,15 @@
     expect(proposal.latitude).to be_nil
     expect(proposal.longitude).to be_nil
   end
+
+  it "remember has_no_address and has_no_image" do
+    fill_proposal(skip_address: true, attach: false)
+
+    click_link "Modify the proposal"
+
+    expect(page).to have_css(".user-device-location button[disabled]")
+    expect(page).to have_css("button.user-device-camera[disabled]")
+  end
 end
 
 shared_examples "4 steps" do
```

Comment:
In this example we should add some cases checking that editing a normal proposal / either draft or the normal one has no changes due the deface overridades.